### ISSUE TITLE
Support for automated testing

### ISSUE THREAD

**Issue by andreas-bulling** [smhaller/ldap-overleaf-sl#7]
Hi,

Do you see a chance to test with/support the latest OSS Sharelatex version (2.6.1 at the moment I believe) and, in general, future latest versions? There is unfortunately no changelog for the things they implement as far as I know. And the open source version is generally put in second place behind the commercial version. But it would still be nice to be current with the small improvements in Sharelatex that they are willing to share with the community.

**Comment by smhaller** [comment]
Lets see if we could do some automated testing with something like https://github.com/rroemhild/docker-test-openldap .

**Comment by andreas-bulling** [comment]
good idea - I guess one line of tests is about whether the changes can still be applied to the updated codebase. A second is what you suggest - whether the changes functionally work with the new Sharelatex version.

**Comment by smhaller** [comment]
I just tried 2.6.1 on my test instance and it seems to work. I updated the repo (but i still didn't look into the CI/CD for the latest versions.. lets see maybe this weekend)

**Comment by haslersn** [comment]
I can confirm that 2.6.1 works; note that I used a [different Dockerfile](https://git.stuvus.uni-stuttgart.de/ref-it/docker-overleaf-ldap/-/commit/7f30e9c476d4afcff00ddbc31be207c231e87215#6651ddff6eb82c840ced7c1dddee15c6e1913dd4) (using `--build-arg BASE=docker.io/sharelatex/sharelatex:2.6.1`).

**Comment by wixaw** [comment]
Hello
I am discovering your LDAP custom. It's great, but I am afraid of the long time! If this is broken I would not want to be blocked with a old version
What do you think ? It's safe ? 

And I would need to tie two branches
Branch1 ou=people for our users (id = cn)(all)
Branch2 ou=peopleext for our partners (id = uid)(memberof=cn=xxxxxx)

It's possible ? 
Thanks

**Comment by smhaller** [comment]
2.6.1 is working. I keep this open to remind that I/we should find a way for automated testing.

For the two branches i think you can write a filter like: (|(FIRSTFILTER)(SECONDFILTER))

**Comment by smhaller** [comment]
3.0.1 is working.
